file.go

```diff
@@ -81,6 +81,7 @@ func extractFiles(zipFilePath, filesToExtractFromZipPath, localPath string) erro
 
 	// Add the path from which we will extract files to the path prefix so we can exclude the appropriate files
 	pathPrefix = filepath.Join(pathPrefix, filesToExtractFromZipPath)
+	pathPrefix = pathPrefix + "/"
```

Comment:
Didn't the `--source-path` arg in #37 already include a trailing slash? Also, will this work on Windows? 

In general, I'm always apprehensive about simple string manipulation and file paths. That's what caused this bug in the first place. 

It seems like the proper solution would be, in the `if strings.Index(f.Name, pathPrefix) == 0` check a few lines below, to convert `f.Name` and `pathPrefix` to canonical paths, and compare those...

Reply:
Since these are zip file paths, the separator is `/` regardless of OS as specified by the `.zip` spec. See section 4.4.17 of https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT.

Reply:
Oh, interesting find!

OK, then one other sanity check on the trailing `/`... How do we know the `--source-path` is a folder?

E.g., If someone passes `--source-path foo`, do we know the user wants the folder `/foo`? Or could it be a file called `foo`? Or could the user want all `/foo*` paths?

Reply:
@yorinasub17 thanks for the link to zip spec. I was nowhere close to thinking about looking for zip spec!

@brikis98 , I tried specifying a file as value for `--source-path`, and `fetch` will download the file. So `fetch` is allowing individual file name to specified as its value. (There is an additional change in `fetch`'s behavior when specifiying a file as value, `fetch` now expects the `<local-download-path>` to be filename not a directory).

Specifying a regular expression such as `/foo*` results in 0 files downloaded. Also, README does not mention allowing regular expressions for `--source-path`, whereas the documentation mentions allowing regular expressions for `--release-asset` option.

Furthermore, not allowing regular expressions for `--source-path` seems to be consistent with the behavior I mentioned above, where we have specify the `<local-download-path>` to be a filename rather than a directory.